### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by cmolinan**
### Changes 
-Creation of a JavaScript array to store the information for all projects.
-Update of the main page so that the projects section is created dynamically using the information stored in that JavaScript object. 
-Implementation of a popup window to show the  project details,

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,39 +12,126 @@
...
+            <li><img src="images/js-tag.png" alt="javaScript"></li>
+            <li></li>
+          </ul>
+          <div class="pop-buttons">
+            <button type="button" class="pop-btn">See Live<img src="images/popup-btn-Icon.png" alt=""></button>
+            <button type="button" class="pop-btn">See Source<img src="images/popup-github.png" alt=""></button>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (ShahierNashaat) [inline index.html:120]: - [ ] Kindly be noticed that the modal popup data is populated with the data from the projects array.
- [ ] So when the user clicks on a project it should load this specific project.

File: cards.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    popup_description: `Lorem Ipsum is simply dummy text of 
+    the printing and typesetting industry. Lorem Ipsum has 
+    been the industry's standard dummy text ever since the 
+    1500s, when an unknown printer took a galley of type 
+    and scrambled it to make a type specimen book. It has
+     survived not only five centuries,but also the leap 
+     into electronic typesetting remaining essent`
+  }
+]  
+}
```
Comment (ShahierNashaat) [inline cards.js:102]: - [ ] Kindly notice that each object in the array contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -1,4 +1,7 @@
 * {
+  /* number of Work Cards */
+  --numCards: 4;
```
Comment (mwanawabangona) [inline styles.css:3]: - [ ] `optional`
- [ ] Good job so far :clap: 
- [ ] Kindly add the blur background as the figma template to all the `desktop and mobile pop-up cards` please screenshots below
- `your template`
Screenshot from 2022-07-22 00-01-33

- `figma template`
Screenshot from 2022-07-21 20-00-11

**Review by mwanawabangona — APPROVED** [review]
Hi @cmolinan,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job getting approved on your first try!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**